fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/MaterializedViewRewriter.java

```diff
@@ -225,6 +228,7 @@ public List<OptExpression> rewrite(ReplaceColumnRefRewriter queryColumnRefRewrit
             if (rewrittenExpression == null) {
                 continue;
             }
+            materializationContext.getOptimizerContext().updateUsedMv(queryTables, materializationContext.getMv());
```

Comment:
no need update here as `updateUsedMv ` is called at `BaseMaterializedViewRewriteRule `

Reply:
done